### DISCUSSION TITLE
Notification for: Category change / Specific category

### DISCUSSION THREAD

**Discussion by Swiftpaws** [Chatterino/chatterino2#4291] [Ideas]
Option per tab to notify on:
* Category change
* Category change (specific category) e.g Notify when category is changed to Minecraft

**Comment by pajlada** [comment]
This feels a bit far fetched for what we do right now.

As a temporary solution you can use something like the TitleChange_Bot https://github.com/zneix/tcb2 (bit technical of a solution) and set up highlights for it

**Comment by x9136** [comment]


**Reply by pajlada** [reply]
What do you mean?
